package.json

```diff
@@ -17,6 +17,7 @@
   ],
   "scripts": {
     "build": "stencil build --docs-json component-docs.json",
+    "build-bindings": "tsc -p tsconfig.bindings.json",
```

Comment:
Modeled after the scripts in the [sample template](
https://github.com/ionic-team/stencil-ds-plugins-demo/blob/04f0108046c02ca494dd6551b1dd822d16b87b8d/packages/component-library-react/package.json#L12-L15).